fdbserver/KeyValueStoreSQLite.actor.cpp

```diff
@@ -2147,7 +2148,8 @@ KeyValueStoreSQLite::KeyValueStoreSQLite(std::string const& filename,
 	                                  diskBytesUsed,
 	                                  freeListPages,
 	                                  id,
-	                                  &readCursors));
+	                                  &readCursors),
+	                       "fdb-sqllite-write");
```

Comment:
I think we need to limit this to 15 characters (the limit in Linux is 16 including null byte). Maybe the name you originally proposed is the best option?

```suggestion
	                       "fdb-sqlite-wr");
```

Reply:
Ah ok, I wasn't sure if it was that simple as we previously discussed these are coroutines and not threads. I suspected there was some indirection going on and we weren't actually going to call `pthread_setname_np` on these. I will fix.

Reply:
Ah, right you are. I guess it's not necessary then to limit the length, so you can decide how you want to do it. Either way, we should change `sqllite` to `sqlite`.